javascript/index.js

```diff
@@ -23,18 +23,185 @@
   getInstruction("mashedPotatoes", 4, (step5) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
     document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
-  }, (error) => console.log(error));
+  }, (error) => console.log(error));*/
 
 
 
 // Iteration 1 - using callbacks
-// ...
+
+getInstruction('mashedPotatoes', 0, (step0) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step0}</li>`
+
+getInstruction('mashedPotatoes', 1, (step1) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step1}</li>`
+
+getInstruction('mashedPotatoes', 2, (step2) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`
+
+getInstruction('mashedPotatoes', 3, (step3) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step3}</li>`
+
+getInstruction('mashedPotatoes', 4, (step4) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step4}</li>`
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>Mashed potatoes are ready!</li>`
+  document.getElementById('mashedPotatoesImg').removeAttribute('hidden');
+
+});});});});});
+
+  getInstruction(0, () => {
```

Comment:
I just wanted to let you know that you don't need this here.
It's better to delete any unwanted/unused code before committing!